zap/src/test/java/org/zaproxy/zap/control/AddOnInstallerUnitTest.java

```diff
@@ -64,7 +65,26 @@ void shouldReturnAddOnLibsDir() throws Exception {
         // Then
         assertThat(
                 Paths.get(Constant.getZapHome()).relativize(addOnLibsDir),
-                is(equalTo(Paths.get("addOnData/addOnId/libs"))));
+                is(equalTo(Paths.get("addOnData/addOnId/1.0.0/libs"))));
+    }
+
+    @Test
+    void shouldDeleteLegacyAddOnLibsDir() throws Exception {
+        // Given
+        AddOn addOnA = new AddOn(createAddOnFile("addOnA.zap"));
+        Path addOnALibsDir = Paths.get(Constant.getZapHome(), "addOnData/addOnA/libs");
+        createFile(addOnALibsDir.resolve("a_a.jar"));
+        createFile(addOnALibsDir.resolve("a_b.jar"));
+        AddOn addOnB = new AddOn(createAddOnFile("addOnB.zap"));
+        Path addOnBLibsDir = Paths.get(Constant.getZapHome(), "addOnData/addOnA/libs");
```

Comment:
Shouldn't the string literal be "B" here?

Reply:
Yes.